.github/workflows/dapr.yml

```
@@ -44,24 +44,22 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest, windows-2019, macOS-latest]
+        os: [ubuntu-latest, macOS-latest]
         target_arch: [arm, arm64, amd64]
         include:
           - os: ubuntu-latest
             target_os: linux
           - os: windows-2019
             target_os: windows
+            target_arch: amd64
+            windows_version: 1809
```

```suggestion
            windows_version: "1809"
```

Or is this not needed? (Otherwise it's parsed as number in YAML)